engines/private/detection.cpp

```diff
@@ -0,0 +1,92 @@
+#include "base/plugins.h"
+#include "engines/advancedDetector.h"
+
+namespace Private {
+static const PlainGameDescriptor privateGames[] = {
+    { "private-eye", "Private Eye" },
+    { 0, 0 }
+};
+
+
+static const ADGameDescription gameDescriptions[] = {
+    {
+        "private-eye",  // US release
+        0,
+        AD_ENTRY1s("ASSETS.Z", "3a7532349cda8126e96dd5e49884af3a", 40232),
```

Comment:
Would it be possible to use another file for detection? I am afraid, `assets.z` is way too generic and exists in many games.

Reply:
I believe I can use `pvteye.z`. That should be better, right?

Reply:
Yes, the chances for file name clash are much lower for this.

Reply:
It should be fixed.